Autopilot/Path Manager/PathManager.c

```diff
@@ -667,12 +667,37 @@ void copyGPSData(){
     pmData.checkbyteDMA2 = generatePMDataDMAChecksum2();
 }
 
+// check if power on reset, brown out reset, watchdog time out or software reset flags are set
+
+ char checkForReset(){
+     if (RCONbits.POR || RCONbits.BOR || RCONbits.WDTO || RCONbits.SWR || RCONbits.EXTR){
+         // clear RCON(Reset control) bits)
```

Comment:
This function might not work.  Or it may break the functionality of StartupErrorCodes.c. When you check these special function registers,  they automatically reset,  as such they can be only read once. 

StartupErrorCodes.c contains get() functions to retrieve this information. Use that instead.